### ISSUE TITLE
withdraw buy sub account

### ISSUE THREAD

**Issue by ntiendat** [gateio/gateapi-python#124]
i need withdraw directly by sub account 
i seted end point address withdraw and full permison for api key but not success
that respose : {"label":"FORBIDDEN","message":"Request API key does not have withdraw permission"}

![image](https://user-images.githubusercontent.com/55078524/179912673-fda5ca11-097a-4083-9410-8690afcd25e3.png)

**Comment by revilwang** [comment]
Duplicate with #123

**Comment by ntiendat** [comment]
> Duplicate with #123

i set full permisson but not success
![aaa](https://user-images.githubusercontent.com/55078524/179913708-aec982d9-e19f-4472-b7c0-6d160cf60a1f.png)

**Comment by revilwang** [comment]
You need to click the square to open the permission

**Comment by ntiendat** [comment]
> You need to click the square to open the permission

that set read and write permisson

**Comment by ntiendat** [comment]
> You need to click the square to open the permission

when set change  this is uncheck

**Comment by ntiendat** [comment]
![image](https://user-images.githubusercontent.com/55078524/179914782-cb2fb77e-f072-4e99-b176-2792e6a5ab87.png)
that check is edit

**Comment by revilwang** [comment]
After the permission is checked. Try again after at most 5 minutes for the permission to take effect

**Comment by ntiendat** [comment]
> After the permission is checked. Try again after at most 5 minutes for the permission to take effect

i wait and try again but not susccess  (this is a sub account)

**Comment by ntiendat** [comment]
> After the permission is checked. Try again after at most 5 minutes for the permission to take effect

i try again after 2h but not success
![image](https://user-images.githubusercontent.com/55078524/179940142-69cbe0e3-8097-4ccd-8004-c1fa9c53e0fb.png)

**Comment by revilwang** [comment]
You can send you API Key(**NOT SECRET**) to dev@mail.gate.io, so we'll have a check.

**Comment by revilwang** [comment]
I've checked the API key's permission. No permission is opened for this API key. 

Do you encounter any problems updating your API key's permission? If your permission is updated successfully, the `Permissions` column in your API key list page should not be empty.

**Comment by ntiendat** [comment]
> I've checked the API key's permission. No permission is opened for this API key.
> 
> Do you encounter any problems updating your API key's permission? If your permission is updated successfully, the `Permissions` column in your API key list page should not be empty.

when i check  withdrawl then permissson change to no permission

**Comment by revilwang** [comment]
I try to edit one of my own keys and the permission can be successfully updated. Are you sure you checked the permission like following?

<img width="524" alt="image" src="https://user-images.githubusercontent.com/8424970/179949498-47ea6d83-10ae-4241-834a-cc58d84fcd8c.png">

**Comment by ntiendat** [comment]
> try to edit one of my own keys and the permission can be successfully updated. Are you sure you checked the permission like following?

yes i edit like you but not oke ( only withdraw permisson), whatever permisson is sucsess
this is api of sub account

**Comment by revilwang** [comment]
I'll report the issue to check if the same thing works for sub account. Sorry for the waiting

**Comment by ntiendat** [comment]
> I'll report the issue to check if the same thing works for sub account. Sorry for the waiting

oke thank you

**Comment by revilwang** [comment]
Sub account is not allowed to perform withdrawal. We'll update the form to exclude withdrawal permission selection in sub account's page.
